iosHackthonApp/Modules/Features/SignUp/Sources/SignUp/SignUpWithEmail.swift

```diff
@@ -9,22 +9,32 @@
 import SwiftUI
 import Strings
 import Components
+import shared
 
 struct SignUpWithEmail: View {
 	@State private var email: String = ""
-	
+	@State private var moveToNextScreen: Bool = false
+
 	var body: some View {
 		VStack(alignment: .leading) {
 			SignUpInfoView(title: Strings.SignUp.signUpWithEmailTitle, description: Strings.SignUp.signUpWithEmailInfo)
 
 			Spacer().frame(maxHeight: .infinity)
 			
 			GeometryReader { geometry in
-				SignUpUserEntryView(isSignupWithEmail: true,
-									textFieldPlaceholder: Strings.SignUp.signUpWithEmailPlaceHolder, buttonTitle: Strings.SignUp.signUpWithEmailButtonTitle,
-									width: geometry.size.width,
-									destinationView: SignUpWithPassword(),
-									entryField: $email)
+				let viewModel = SignUpUserEntryViewModel(
+					isSignupWithEmail: true,
+					textFieldPlaceholder: Strings.SignUp.signUpWithEmailPlaceHolder,
+					buttonTitle: Strings.SignUp.signUpWithEmailButtonTitle,
+					width: geometry.size.width,
+					entryField: $email,
+					signUp: self.signUp,
+					moveToNextScreen: $moveToNextScreen,
+					isLoading: .constant(false))
```

Comment:
might be wrong on this one, but should be able to set an initial value of false in the viewModel

Reply:
Added  the value to the viewmodel